sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/RegistryArtifactBase.java

```diff
@@ -1,101 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Was this an Impl file and we don't need it anymore? 
I see we moved the content to respective sync/async clients. 
Just wanted to know if this was ever a autorest generated file.

Reply:
good catch! it wasn't auto-generated, I guess the header was copied :)
I removed the generated note from couple of other files were is was incorrectly set.